build.sbt

```diff
@@ -243,6 +259,28 @@ lazy val interfaces = project
     ),
   )
 
+lazy val mtagsShared = project
+  .in(file("mtags-shared"))
+  .settings(sharedSettings)
+  .settings(
+    moduleName := "mtags-shared",
+    crossTarget := target.value / s"scala-${scalaVersion.value}",
+    crossScalaVersions := {
+      V.supportedScalaVersions ++ V.nightlyScala3Versions
+    },
+    crossVersion := CrossVersion.binary,
+    Compile / packageSrc / publishArtifact := true,
```

Comment:
That's the default in the build, can be removed

Reply:
This was overriden here:
https://github.com/rochala/metals/blob/ef7e82144bd0f6f3baf0052e1088de9e4861aa3e/build.sbt#L66

Reply:
Wait, that would mean they are overriden to true on CI, where it will be released. Not locally, sure.

Reply:
Yea, but when you are working with metals, it will be very annoying to add this line every time you want to deploy mtags-shared locally, just to test if something works in dotty.